books/free-programming-books-da.md

```diff
@@ -31,3 +31,7 @@
 ### Pascal
 
 * [Programmering i Pascal](http://people.cs.aau.dk/~normark/all-basis-97.pdf) - Kurt Nørmark (PDF)
+
+### Java
```

Comment:
### Java category(heading) is already present on LINE 26, add your course there. Thanks